app/models/Questionnaire.scala

```diff
@@ -20,8 +20,8 @@ import play.api.data.Form
 import play.api.data.Forms.{mapping, number, optional, text}
 
 case class Questionnaire(
-                          easyToUse: Option[Int],
-                          satisfactionLevel: Option[Int],
+                          easyToUse: Int,
```

Comment:
Reason for these two to no longer be optional?

Reply:
I noticed users could send a blank feedback form so I made the easyToUse and satisfcationLevel (radio) fields mandatory, returning a field validation error to the user if they don't. I've shared this with the design team and they're looking into whether or not its necessary.